### PR TITLE
move metadata() to Item in order to support folder metadata

### PR DESCRIPTION
**Pull request by sp4x**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @sp4x, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: boxsdk/object/item.py [hunk 1]
```diff
@@ -322,3 +323,23 @@ def delete(self, params=None, etag=None):
         """
         headers = {'If-Match': etag} if etag is not None else None
         return super(Item, self).delete(params, headers)
+
+    def metadata(self, scope='global', template='properties'):
+        """
+        Instantiate a :class:`Metadata` object associated with this file.
```
Comment (Jeff-Meadows) [inline boxsdk/object/item.py]: comment should now specify `item` or `file or folder`

File: boxsdk/object/item.py [hunk 2]
```diff
@@ -322,3 +323,23 @@ def delete(self, params=None, etag=None):
...
+            Scope of the metadata. Must be either 'global' or 'enterprise'.
+        :type scope:
+            `unicode`
+        :param template:
+            The name of the metadata template.
+            See https://box-content.readme.io/reference#metadata-object for more details.
+        :type template:
+            `unicode`
+        :return:
+            A new metadata instance associated with this file.
```
Comment (Jeff-Meadows) [inline boxsdk/object/item.py]: comment should now specify `item` or `file or folder`

**Comment by Jeff-Meadows** [comment]
👍 

Please update the comments and then this looks ready to merge once the CLA bot is happy.

**Comment by sp4x** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @sp4x has just signed the CLA. Thanks, and we look forward to your contribution.

**Comment by Jeff-Meadows** [comment]
Going to close this since I just opened #167, and this one has merge conflicts. Thanks for the PR and I've added you to our list of contributors in #167.

**Pull request closed without merging**